armi/settings/fwSettings/globalSettings.py

```diff
@@ -158,6 +159,14 @@ def defineSettings() -> List[setting.Setting]:
             "block. Used for axial mesh refinement.",
             schema=vol.All(vol.Coerce(int), vol.Range(min=0, min_included=False)),
         ),
+        setting.Setting(
+            CONF_UNIFORM_MESH_MINIMUM_SIZE,
```

Comment:
when it is left as None, i think it goes back to the old behavior of not preserving fuel boundaries as well, so it might be worthwhile to call out that setting any number here will do a better job of preserving material boundaries.